README.md

```diff
@@ -1,13 +1,12 @@
 # IMUV - Flying Campus
 
 ![IMUV-Homepage](./img/IMUV_Homepage.png)
-Online demo : https://www.imuvirtuel.fr/.
+Online demo: https://www.imuvirtuel.fr/.
 
 IMUV is a game application built on the [UD-Viz](https://github.com/VCityTeam/UD-Viz) framework.
 
-It's composed of two parts: the server and the client.
-`client` is a [front-end](https://en.wikipedia.org/wiki/Front_end_and_back_end) application requiring `server`
-its [back-end](https://en.wikipedia.org/wiki/Front_end_and_back_end) node application providing an http server and a websocket communication layer (expecting Imuv client connections).
+## A client-server application
```

Comment:
client no longer exists idem for server. it could be something like: Imuv is a web application composed of a backend and a front end. backend code can be found in ./bin/backend/ and in src/shared, front end code in src/browser and also src/shared